CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ All notable changes to **Pipecat** will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+
+- Add support for more languages to ElevenLabs (arabic, croatian, filipino,
```

Comment:
```suggestion
- Add support for more languages to ElevenLabs (Arabic, Croatian, Filipino,
```

Reply:
Let's capitalize all of these languages.

Reply:
Done!